pnpm-lock.yaml

```diff
@@ -3122,9 +3122,9 @@ packages:
     dependencies:
       '@nuxt/types': 2.15.8_webpack@4.46.0
       consola: 2.15.3
-      fork-ts-checker-webpack-plugin: 6.5.0_mpahngmn3mtdx6vmys44ywifxm
-      ts-loader: 8.3.0_7in6qbnjgkqokibdcer35wqugu
-      typescript: 4.6.2
+      fork-ts-checker-webpack-plugin: 6.5.0_74f2hvla7n4b757hwnerd4quka
+      ts-loader: 8.3.0_e7hrjdrs22zc4syxbltzlwluhe
+      typescript: 4.2.4
```

Comment:
:scream: 

Looks like we still need that TypeScript resolution after all :confused: